design/design-doc-stacks.md

```diff
@@ -495,14 +495,19 @@ spec:
 
 #### Stack CRD Labels
 
+Labels and annotations provided in the source CRD are preserved, such as
+`controller-tools.k8s.io` in the example above.
+
 The labels added by Crossplane tools serve the following purpose:
 
 Label | Value | Purpose
 ---|--- | ---
 `app.kubernetes.io/managed-by` | `stack-manager` | This [recommended label](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) identifies the tool being used to manage the operation of the application.
+`core.crossplane.io/parent-kind` | `{kind}.{apiGroup}` | The kind of the Stack ("stack.stacks.crossplane.io")
```

Comment:
I am not sure whether it makes more sense to use `parent-kind` or `parent.kind`, but I believe it's worth considering. I think this is probably a question for someone with more prior experience in Kubernetes land, such as @jbw976

Reply:
I think going with hyphenated labels follows the common theme taken by k8s in many of their recommended and common labels so I think it works but I will defer to @jbw976 for the call.

Reply:
Yes, hyphens is what is fairly broadly used in the ecosystem, that sounds good to me